### PR TITLE
Minor updates to content

### PR DESCRIPTION
**Pull request by Srushtika**
Please review before merging. Had done changes a few days back while reading through docs, forgot to raise a PR.

### CODE REVIEW

File: content/general/authentication.textile [hunk 1]
```diff
@@ -30,7 +30,7 @@ Every application has one or more private API keys that you set up and "configur
 </a>
 
 **Using Token Authentication**
-Client-side devices should generally be considered untrusted, and as such, it is important that you minimize the impact of any credentials being compromised on those devices. The recommended strategy therefore for authenticating client-side devices is to issue them with time-limited tokens with only the privileges (capabilities) that are needed for that device. It is therefore the responsibility of the developer to identify each device before issuing a token. In the example below, we show you a a typical authentication workflow for client-side devices.
+Client-side devices should generally be considered untrusted, and as such, it is important that you minimize the impact of any credentials being compromised on those devices. Therefore, the recommended strategy for authenticating client-side devices is to issue them with time-limited tokens with only the privileges (capabilities) that are needed for that device. It is therefore the responsibility of the developer to identify each device before issuing a token. In the example below, we show you a a typical token authentication workflow for client-side devices.
```
Comment (tomczoink) [inline content/general/authentication.textile]: " we show you a a typical token " => "we show you a typical token". (not part of your changes but good to fix!)

File: content/types/_client_options.textile [hunk 2]
```diff
@@ -20,9 +20,9 @@ h4.
...
 
-- <span lang="default">tokenDetails</span><span lang="csharp">TokenDetails</span><span lang="python">token_details</span><span lang="ruby">:token_details</span> := An authenticated "@TokenDetails@":/realtime/types/#token-details object that is most commonly obtained from of a token request response. Use this option if you wish to use Token authentication. Read more about "Token authentication":/general/authentication#token-authentication<br>__Type: @TokenDetails@__
+- <span lang="default">tokenDetails</span><span lang="csharp">TokenDetails</span><span lang="python">token_details</span><span lang="ruby">:token_details</span> := An authenticated "@TokenDetails@":/realtime/types/#token-details object that is most commonly obtained from of a token request's response. Use this option if you wish to use Token authentication. Read more about "Token authentication":/general/authentication#token-authentication<br>__Type: @TokenDetails@__
```
Comment (tomczoink) [inline content/types/_client_options.textile]: Again not your change, but "that is most commonly obtained from of a token request's response" => "that is most commonly obtained from a token request's response".

File: content/types/_client_options.textile [hunk 3]
```diff
@@ -20,9 +20,9 @@ h4.
...
 
-- <span lang="default">tls</span><span lang="csharp">Tls</span><span lang="ruby">:tls</span> := _true_ A boolean value, indicating whether or not a TLS ("SSL") secure connection should be used. An insecure connection cannot be used with Basic authentication ensuring private keys are compromised in transit. "Find out more about TLS":https://support.ably.io/solution/articles/3000045208-are-messages-sent-to-and-received-from-ably-sent-security-using-tls<br>__Type: @Boolean@__
+- <span lang="default">tls</span><span lang="csharp">Tls</span><span lang="ruby">:tls</span> := _true_ A boolean value, indicating whether or not a TLS ("SSL") secure connection should be used. An insecure connection cannot be used with Basic authentication as it would lead to a possilble compromise of the private API key while in transit. "Find out more about TLS":https://support.ably.io/solution/articles/3000045208-are-messages-sent-to-and-received-from-ably-sent-security-using-tls<br>__Type: @Boolean@__
```
Comment (tomczoink) [inline content/types/_client_options.textile]: "possilble" => "possible".

File: content/general/authentication.textile (hunk 1 above)
Comment (Srushtika) [reply content/general/authentication.textile]: Good find, I missed it :) Thanks!

File: content/types/_client_options.textile (hunk 2 above)
Comment (Srushtika) [reply content/types/_client_options.textile]: Fixed.

File: content/types/_client_options.textile (hunk 3 above)
Comment (Srushtika) [reply content/types/_client_options.textile]: Fixed.

**Pull request merged**